### ISSUE TITLE
Declaration of jsonSerialize() must be compatible

### ISSUE THREAD

**Issue by mialyGit** [benjaminhirsch/nova-slug-field#35]
![Fix it](https://github.com/tomhatzer/nova-slug-field/assets/50203282/72e2e8cf-1d28-4d86-b039-6f115b92e253)

**Comment by cyriljulien** [comment]
For the overload (or implementation of an interface) of a method to be valid, the signatures of the methods must be identical. Here, the jsonSerialize() method of the BenjaminHirsh/NovaSlugField/slug class must have the exact same signature as that of the Laravel/Nova/Fields/Field class.

To resolve this error, check the declaration of the jsonSerialize() method in the BenjaminHirsh/NovaSlugField/slug class and ensure that it has the same signature as that of the parent class Laravel/Nova/Fields/Field.

**Comment by cyriljulien** [comment]
I replace with this :

```
public function jsonSerialize() : public function jsonSerialize() : array
    {
        return array_merge([
            'disableAutoUpdateWhenUpdating' => $this->disableAutoUpdateWhenUpdating,
            'slugifyOptions' => $this->slugifyOptions,
            'showPreviewUrl' => $this->showUrlPreview,
            'slugPrefix' => $this->slugPrefix,

        ], parent::jsonSerialize());
    }
    {
        return array_merge([
            'disableAutoUpdateWhenUpdating' => $this->disableAutoUpdateWhenUpdating,
            'slugifyOptions' => $this->slugifyOptions,
            'showPreviewUrl' => $this->showUrlPreview,
            'slugPrefix' => $this->slugPrefix,

        ], parent::jsonSerialize());
    }

**Comment by tomhatzer** [comment]
PR #36 has been merged and a new tag (1.4.0) has been released.
